### PR TITLE
Introduce new workflow example that demonstrates how design data crea…

### PR DESCRIPTION
**Pull request by yeleva**
…ted by machine control systems running in the field can be downloaded and converted to landxml. This example supersedes download_operator_pt3_files_as_landxml

### CODE REVIEW

File: workflows/download_device_design_data_as_landxml/download_device_design_data_as_landxml.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+def particular_from_design_type(a_design_type):
+    if(a_design_type == "Lines"):
+        return "LN3"
+    elif(a_design_type == "Planes"):
+        return "PL3"
+    elif(a_design_type == "Points"):
+        return "PT3"
+    elif(a_design_type == "Roads"):
+        return "RD3"
+    else:
```
Comment (topcon-lrietberg) [inline workflows/download_device_design_data_as_landxml/download_device_design_data_as_landxml.py:36]: missing particular "TN3" for design type "Surfaces"

File: workflows/download_device_design_data_as_landxml/download_device_design_data_as_landxml.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        converted_dir = os.path.join(output_dir, "converted")
+        rdm_file_name = fi["value"]["name"]
+
+        os.makedirs(original_dir, exist_ok=True)
+        os.makedirs(converted_dir, exist_ok=True)
+
+        particular = particular_from_design_type(fi["value"]["designType"])
+        base = "{0}/designfile/v1/sites/{1}/design_files/{2}?design_type={3}".format(server.to_url(), args.site_id, fi["value"]["doFileUUID"], fi["value"]["designType"])
+        url_original = ["{}&particular={}".format(base, particular), particular.lower(), original_dir]
+        url_landxml  = ["{}&particular=LANDXML".format(base, fi["value"]["designType"]), "xml", converted_dir]
```
Comment (topcon-lrietberg) [inline workflows/download_device_design_data_as_landxml/download_device_design_data_as_landxml.py:85]: format string "{}&particular=LANDXML" only contain one input (design type is not needed here)

File: workflows/download_device_design_data_as_landxml/download_device_design_data_as_landxml.py (hunk 1 above)
Comment (yeleva) [reply workflows/download_device_design_data_as_landxml/download_device_design_data_as_landxml.py:36]: Thank you.

File: workflows/download_device_design_data_as_landxml/download_device_design_data_as_landxml.py (hunk 2 above)
Comment (yeleva) [reply workflows/download_device_design_data_as_landxml/download_device_design_data_as_landxml.py:85]: I get a 400 when doing this.

**Pull request merged**